### PR TITLE
Helm Chart: Add Flux git label and registry options 

### PR DESCRIPTION
**Pull request by stefanprodan**

### CODE REVIEW

**Comment by stefanprodan** [comment]
@squaremo I've set the `ciSkip=true` to match the Weave Cloud defaults.

**Comment by squaremo** [comment]
I really don't like the chart having its own defaults; especially where they differ from the actual defaults. The only reason for them to be different in the chart is if it's necessary for running with helm.

File: chart/flux/README.md [hunk 1]
```diff
@@ -161,9 +161,18 @@ The following tables lists the configurable parameters of the Weave Flux chart a
 | `git.path` | Path within git repo to locate Kubernetes manifests (relative path) | None
 | `git.user` | Username to use as git committer | `Weave Flux`
 | `git.email` | Email to use as git committer | `support@weave.works`
+| `git.setAuthor` | If set, the author of git commits will reflect the user who initiated the commit and will differ from the git committer. | `false`
+| `git.label` | Label to keep track of sync progress, used to tag the Git branch | `flux-sync`
+| `git.ciSkip` | Append "[ci skip]" to commit messages so that CI will skip builds | `false`
 | `git.chartsPath` | Path within git repo to locate Helm charts (relative path) | `charts`
```
[the remark this answers is not in the document]
Comment (stefanprodan) [reply chart/flux/README.md]: It is in the operator as well. I didn't want to break the Helm upgrade, but I can remove it and bump the chart major version.

File: chart/flux/README.md [hunk 2]
```diff
@@ -161,9 +161,18 @@ The following tables lists the configurable parameters of the Weave Flux chart a
...
 | `git.pollInterval` | Period at which to poll git repo for new commits | `30s`
```
[the remark this answers is not in the document]
Comment (stefanprodan) [reply chart/flux/README.md]: Yes this was set to 30s from start when I first created the chart. I've used it in my demos and I forgot to change it back.

File: chart/flux/README.md [hunk 3]
```diff
@@ -161,9 +161,18 @@ The following tables lists the configurable parameters of the Weave Flux chart a
...
-| `ssh.known_hosts`  | The contents of an SSH `known_hosts` file, if you need to supply host key(s) |
+| `ssh.known_hosts`  | The contents of an SSH `known_hosts` file, if you need to supply host key(s) | None
+| `registry.cacheExpiry` | Duration to keep cached image info in memcached | `1h`
+| `registry.pollInterval` | Period at which to check for updated images | `5m`
+| `registry.rps` | Maximum registry requests per second per host | `200`
+| `registry.burst` | Maximum number of warmer connections to remote and memcache | `125`
+| `registry.trace` |  Output trace of image registry requests to log | `false`
+| `registry.insecureHosts` | Comma-separated value of HTTP registry domains | None
```
[the remark this answers is not in the document]
Comment (squaremo) [reply chart/flux/README.md]: "Use HTTP rather than HTTPS for these image registry domains" is closer to the arg description

**Comment by squaremo** [comment]
Don't worry about the things I commented on but weren't actually changes here.

**Comment by squaremo** [comment]
Do you need to mention the breaking change (removing git.chartPath) anywhere?

**Comment by stefanprodan** [comment]
We could mention it maybe in the release notes of helm-op

**Pull request merged**